static/src/javascripts/projects/common/modules/ui/first-pv-consent-plus-engagement-banner.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+import { Message } from 'common/modules/ui/message';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+import type { Banner } from 'common/modules/ui/bannerPicker';
+import { acquisitionsBannerControlTemplate } from 'common/modules/commercial/templates/acquisitions-banner-control';
+import { engagementBannerParams } from 'common/modules/commercial/membership-engagement-banner-parameters';
+import { addTrackingCodesToUrl } from 'common/modules/commercial/acquisitions-ophan';
+import {
+    messageCode as engagementMessageCode,
+    canShow as canShowEngagementBanner,
+} from 'common/modules/commercial/membership-engagement-banner';
+import {
+    track as trackFirstPvConsent,
+    bindClickHandlers as bindFirstPvConsentClickHandlers,
+    canShow as canShowFirstPvConsent,
+    messageCode as firstPvConsentMessageCode,
+    makeHtml as makeFirstPvConsentHtml,
+    template as firstPvConsentTemplate,
+    bindableClassNames as firstPvConsentBindableClassNames,
+} from 'common/modules/ui/first-pv-consent-banner';
+import marque36icon from 'svgs/icon/marque-36.svg';
+
+const messageCode: string = 'first-pv-consent-plus-engagement-banner';
+
+const bannerParams: EngagementBannerParams = engagementBannerParams(
+    geolocationGetSync()
+);
+
+const bannerTemplateParams: EngagementBannerTemplateParams = {
+    messageText: bannerParams.messageText,
+    ctaText: bannerParams.ctaText,
+    buttonCaption: bannerParams.buttonCaption,
+    linkUrl: addTrackingCodesToUrl({
+        base: bannerParams.linkUrl,
+        componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
+        componentId: 'first_pv_consent_plus_support_the_guardian_banner',
+    }),
+};
+
+const bannerHtml = `
+    <div class="site-message js-site-message js-double-site-message site-message--banner site-message--double-banner" tabindex="-1" role="dialog" aria-label="welcome" aria-describedby="site-message__message" data-component="AcquisitionsEngagementBannerStylingTweaks_control">
+        <div class="js-engagement-banner-site-message site-message--engagement-banner">
```

Comment:
Are these banners dismissable for user's using keyboard navigation? The negative `tabindex` must mean the banner is not reachable via key navigation?

Reply:
This is copied from the template in `message.scala.html` so it's true of all banners